### PR TITLE
Add crud operations

### PR DESCRIPTION
**Pull request by Techtrial101**
# Add crud operations

- Add Edit button to each BlogDetails component
- Implement handleDelete using axios.delete in BlogDetails.js
- Implement handleSubmit using 'POST'  and  'PUT' method in Create.js to add new component to the Server
- Add useNavigate hook to route back to homepage after each Submit and Delete 
- Add License and Updated Readme.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,85 +1,200 @@
...
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-### `npm run eject`
+<!-- LIVE DEMO -->
 
-**Note: this is a one-way operation. Once you `eject`, you can't go back!**
+## 🚀 Live Demo <a name="live-demo"></a>
 
-If you aren't satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
+### [Webloaded]()
```
Comment (Bestbynature) [inline README.md:64]: @Techtrial101 , you actually need to include the live demo link on lines 29 and 63, at the very least. The link is obtained from the deployment process which has been explained in the video. If you have difficulty getting it, post in the group and ask a colleague to help.

File: data/wb.json [hunk 2]
```diff
@@ -1,64 +1,76 @@
...
+      "body": "This is my second creator post",
+      "author": "John fresco",
+      "id": "2"
+    },
+    {
+      "title": "English",
+      "body": "This is my third creator post",
+      "author": "Elizabeth florish",
+      "id": "3"
+    },
```
Comment (Bestbynature) [inline data/wb.json]: The ids in your json file are currently in string format. Kindly check the message i created on the whatsapp group to fix this.

File: src/App.js [hunk 3]
```diff
@@ -35,13 +28,13 @@ function App() {
         <Route path='/' element={<Home/>} />
         <Route path='/create' element={<Create/>} />
         <Route path='*' element={<NotFound />}/>
+        <Route path="/edit/:id" element={<Create/>}/>
         <Route path= "/creators/:id" element={<BlogDetail/>}/>
         </Routes>
 
       
         <Navbar/>
```
Comment (Bestbynature) [inline src/App.js]: The Navbar component should be above all the Routes. So it should be on line 26

File: src/Navbar.js [hunk 4]
```diff
@@ -11,10 +11,13 @@ function handleClick(){
 }
     return (  
         <div className="navbar">
+            <img src="webl.png" alt="React Logo" className="logo"/>
```
Comment (Bestbynature) [inline src/Navbar.js]: This image should link back to the home directory or the '/' directory.

**Review by Bestbynature — CHANGES_REQUESTED** [review]
Almost There GIF


Hello @Techtrial101 , Although you have given so much effort to get this work submitted, and you're probably tired, you still need to do a few more things before moving on to the next milestone.

Please check the comments under review.

File: README.md (hunk 1 above)
Comment (Techtrial101) [reply README.md:64]: I apologize for the inconvenience caused. As per your request, I created a new account yesterday and i was told it needed verification. when i didn't get anything i created a new one  today to resolve the issue with the previous account. However, I encountered an unexpected error as I did not receive any verification code for the new account. I am currently unable to access my Vercel account to check for the code, but once it becomes available, I will ass my links

File: data/wb.json (hunk 2 above)
Comment (Techtrial101) [reply data/wb.json]: I have repeated the process several times and up till now it keeps bringing string after I have removed them.

File: src/App.js (hunk 3 above)
Comment (Techtrial101) [reply src/App.js]: I have done this sir

File: src/Navbar.js (hunk 4 above)
Comment (Techtrial101) [reply src/Navbar.js]: I have done this sir

File: README.md (hunk 1 above)
Comment (Bestbynature) [reply README.md:64]: okay @Techtrial101 , ass your link as you have said before the next submission.

**Review by Bestbynature — APPROVED** [review]
# APPROVED

```
I am currently unable to access my Vercel account to check for the code, but once it becomes available, I will ass my links
```

Please ensure to ass your link as promised. 😄

**Pull request merged**